src/YAxis.re

```diff
@@ -8,49 +8,155 @@ external make:
     ~allowDataOverflow: bool=?,
     ~allowDecimals: bool=?,
     ~allowDuplicatedCategory: bool=?,
-    ~axisLine: 'axisLine=?,
+    ~axisLine:
+      [@mel.unwrap] [
+        | `Bool(bool)
+        | `Obj(Js.t({..}))
+      ]
+        =?,
     ~className: string=?,
-    ~dataKey: string=?,
+    ~dataKey:
+      [@mel.unwrap] [
+        | `Str(string)
+        | `Int(int)
+        | `Fn('dataObj => 'data)
+      ]
+        =?,
     ~domain: array('domain)=?,
     ~height: int=?,
     ~hide: bool=?,
     ~interval: AxisInterval.t=?,
-    ~label: 'label=?,
+    ~label:
+      [@mel.unwrap] [
+        | `Str(string)
+        | `Int(int)
+        | `Float(float)
+        | `Component(React.element)
```

Comment:
I'd call this `Element`. (A component is what creates elements, aka the `make` function). Same for other usages in other modules.

Reply:
Done

Reply:
Ah... I remember now that I named it that to be consistent with [this](https://github.com/ahrefs/melange-recharts/blob/master/src/Utils.re#L162). I feel like I should also update this as well.